syntax = "proto3";
    
service DataEngine {
    rpc GiveData (DataRequest) returns (DataResponse) {}
}
  
message DataRequest {
    string hashtag = 1;
    string limit = 2;
}

message DataResponse {
    repeated DataTweet tweet = 1;
    int32 lenght = 2;
}
  
message DataTweet {
    string id = 1;
    string username = 2;
    string created_at = 3;
    string tweet = 4;
    string name = 5;
    repeated string hashtags = 6;
}

